posts/Yoann/2016-12-20-hibernate-search-5-6-0-CR1-and-5-7-0-Beta2.adoc

```diff
@@ -0,0 +1,99 @@
+= First candidate release for Hibernate Search with Elasticsearch!
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+It's finally here! We just published the first candidate release of Hibernate Search with experimental Elasticsearch integration (5.6.0.CR1), along with 5.7.0.Beta2.
+
+Version 5.6.0.CR1 brings the https://hibernate.atlassian.net/issues/?jql=fixVersion%20%3D%205.6.0.CR1[latest bugfixes and previously missing features] for our experimental Elasticsearch integration. This is the version to use with Hibernate ORM versions 5.0.x and 5.1.x.
+
+Version 5.7.0.Beta2 brings the exact same changes as 5.6.0.CR1, and it still features the compatibility with Hibernate ORM
+version 5.2.x that was introduced with 5.7.0.Alpha1.
+
+== What's new?
+
+ * https://hibernate.atlassian.net/browse/HSEARCH-2348[HSEARCH-2348]: Elasticsearch queries are now logged at the
+   `DEBUG` level on the `org.hibernate.search.fulltext_query` logger, just as embedded Lucene queries.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2449[HSEARCH-2449]: more configuration options have been added regarding
+   the connection to Elasticsearch, including the size of connection pools, the connection/read timeouts, and
+   the automatic discovery of new nodes in the Elasticsearch cluster for client-side loadbalancing and (limited) failover.
+ * https://hibernate.atlassian.net/browse/HSEARCH-2219[HSEARCH-2219]: analyzer definitions are now automatically
+   translated and added to the Elasticsearch settings when Elasticsearch indexes are created.
+   Please note there are still some limitations, most notably the definitions are not updated automatically (no support
+   for the MERGE schema management strategy).
+   See the https://docs.jboss.org/hibernate/search/5.6/reference/en-US/html_single/#elasticsearch-mapping-analyzer[documentation]
+   for more information about analyzers with Elasticsearch.
+ * ... and much more, mainly bug fixes.
+   The full change log can be found on https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10061&version=25900[our JIRA instance]
+   or on https://github.com/hibernate/hibernate-search/blob/5.6.0.CR1/changelog.txt[our GitHub repository].
+
+== When will this 5.6.0 be released?
+
+As far as we are concerned, 5.6.0 is ready.
```

Comment:
I find it weird that you use 5.6.0 as version, if you prefer to be precise you should use 5.6.0.Final otherwise I think 5.6 would be enough.